pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -122,7 +122,7 @@ func (tc *Controller) Run(workers int, stopCh <-chan struct{}) {
 
 func (tc *Controller) addJob(obj interface{}) {
 	job := obj.(*batch.Job)
-	klog.V(4).Infof("Adding job %s/%s", job.Namespace, job.Name)
+	klog.V(4).InfoS("Adding Job", "jobNamespace", job.Namespace, "job", job.Name)
```

Comment:
```suggestion
	klog.V(4).InfoS("Adding Job", "jobNamespace", job.Namespace, "jobName", job.Name)
```

Reply:
This can actually use `klog.KRef`, thanks for catching it.